common/src/main/java/eu/europeana/cloud/common/model/dps/StatisticsReport.java

```diff
@@ -0,0 +1,44 @@
+package eu.europeana.cloud.common.model.dps;
+
+import javax.xml.bind.annotation.XmlRootElement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Model class for statistics report.
+ */
+@XmlRootElement()
+public class StatisticsReport {
+
+    private long id;
```

Comment:
Maybe this member should be named in a more informative way? Id can mean anything...